tests/iio_attr.c

```diff
@@ -367,26 +471,27 @@ static void usage(void)
 				options_descriptions[i]);
 		if (i == 3)
 			printf("Optional qualifiers:\n");
-		if (i == 6)
+		if (i == 8)
 			printf("Attribute types:\n");
 	}
 }
 
 int main(int argc, char **argv)
 {
 	struct iio_context *ctx;
-	int c, option_index = 0;
+	int c, option_index = 0, errnum;
 	int device_index = 0, channel_index = 0, attr_index = 0;
-	const char *arg_uri = NULL;
+	const char *arg_uri = NULL, *gen_file = NULL;
```

Comment:
`arg_uri` is always NULL

Reply:
Just keeping with the rest of the coding style.